toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -85,8 +85,25 @@ func CreateInstallRoot(installRoot string, mountPointMap, mountPointToMountArgsM
 	}
 	installMap[rootMountPoint] = mountPointMap[rootMountPoint]
 
-	// Mount rest of the mountpoints
-	for mountPoint, device := range mountPointMap {
+	// Convert the installMap into a slice of mount points so it can be sorted
+	var remainingMounts []string
+	for mountPoint := range mountPointMap {
+		// Skip empty mount points
+		if mountPoint == "" {
```

Comment:
>if mountPoint == "" [](start = 2, length = 19)

Do you know why that's even allowed? Shouldn't we error out at some point?

Reply:
No idea, probably should be caught in a IsValid() fuction for PartitionSetting structs. Currently that is a stubbed function.

---
In reply to: [550320651](https://github.com/microsoft/CBL-Mariner/pull/481#discussion_r550320651) [](ancestors = 550320651)